src/httpapi.cpp

```diff
@@ -16,15 +25,18 @@ HTTPSClient httpsClients[POOL_SIZE];
 
 HTTPAPI_RESULT HTTPAPI_Init(void)
 {
+    #if defined(ARDUINO_ARCH_SAMD)
```

Comment:
I know this isn't a great spot to setup the ```NTPClient``` (maybe there's a better spot), but it would be nice to sync the NTP time on the ESP8266 and all supported platforms in the same place..